cmd/gamecommand/gamecommand.go

```diff
@@ -0,0 +1,56 @@
+package gamecommand
+
+import (
+	"bufio"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/UnnecessaryRain/ironway-core/pkg/game"
+	"github.com/UnnecessaryRain/ironway-core/pkg/game/commands"
+	log "github.com/sirupsen/logrus"
+	kingpin "gopkg.in/alecthomas/kingpin.v2"
+)
+
+type gameCommand struct {
+	player string
+}
+
+// Configure sets up the command for gamer
+func Configure(app *kingpin.Application) {
+	g := &gameCommand{}
+	c := app.Command("game", "starts a game").
+		Action(g.run)
+	c.Flag("player", "player to use for this game").
+		Short('p').
+		Default("irony42").
```

Comment:
What this? Plain text username or password? Some environment var?
Can you tag with a `//FIXME` or `//HACK` or something?